### ISSUE TITLE
Downlading firmware xxx-template/firmware.bin failed: 404

### ISSUE THREAD

**Issue by neo799** [esphome/esphome-project-template#5]
Hi,

I set up a repository for my RGB Controller template but somhow it does ot work.
https://github.com/Planet-Laterna/Laterna-ESPHome-template
The github workflow works fine and it compiles everything.

When I try to install the firmware via https://planet-laterna.github.io/Laterna-ESPHome-template/ I get the following error message:
Downlading firmware laterna-template/firmware.bin failed: 404

I checked the github pages and it seems like the name of the project as in .github/workflows/publish.yml is added to the folder name for the firmware. That looks different to the one here in the template repository.

Can you please check?

**Comment by jesserockz** [comment]
Yes it seems I broke something in the workflow. I will take a look, thanks for reporting.
